DiamondLightSource/SynchWeb
# LIMS-1108: Improve message re international shipping

**JIRA ticket**: [LIMS-1108](https://jira.diamond.ac.uk/browse/LIMS-1108)

**Summary**:

Improve the message displayed regarding international shipping

**Changes**:
- Add a new variable for a link to help with international shipping `$facility_courier_countries_link`
- If that variable exists, display the message at the top of the add shipment page, with the link
- Change the message displayed at the top of a shipment page depending on whether the dewar's country is supported for Synchweb-based shipping (ie UK), semi-supported (ie EU), or not supported.

**To test**:
- Click to add a new shipment, check no red message is displayed at the top
- Add `$facility_courier_countries_link = 'https://www.diamond.ac.uk/Instruments/Mx/I03/I03-Manual/Shipping-Samples.html';` to config.php, refresh and check a red message appears at the top with a link
- View a booked shipment from the UK, check it shows a message about printing the air waybill
- View an unbooked shipment from the UK, check it shows a message about booking the air waybill
- View a shipment from the EU, check it shows a message including a link to help for EU users
- View a shipment from outside the EU, check it shows a shorter message without the EU link
